content/security/advisory/2015-11-11.adoc

```diff
@@ -60,7 +60,7 @@ API tokens of other users were exposed to admins by default. On instances that d
 === JNLP agents not subject to agent-to-controller access control
 *SECURITY-206 / CVE-2015-5325*
 
-Agents connecting via JNLP were not subject to the optional agent-to-controller access control documented at Link:/security-144 (CVE-2014-3665).
+Agents connecting via JNLP were not subject to the optional agent-to-controller access control documented at link:/security-144 (CVE-2014-3665).
```

Comment:
Link doesn't render correctly without this change.

```suggestion
Agents connecting via JNLP were not subject to the optional agent-to-controller access control documented at link:/security/advisory/2014-10-30/[(CVE-2014-3665)].
```

## Before the change:

![image](https://user-images.githubusercontent.com/156685/116632126-a4d7a200-a913-11eb-8e9b-7a8fda1eb841.png)

## After the change

![image](https://user-images.githubusercontent.com/156685/116633035-b15cfa00-a915-11eb-89eb-10e9d6c163fe.png)

If that's not what is intended, then maybe it should be `link:/redirect/security-144[(CVE-2014-3665)]` ?

Reply:
Restored the URL.